src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -529,6 +533,12 @@ public MockResponse dispatch(RecordedRequest request) {
                                 .setBody(mapper.writeValueAsString(lineAttachResponseInfos))
                                 .addHeader("Content-Type", "application/json; charset=utf-8");
                     }
+                } else if ("DELETE".equals(request.getMethod()) && path.contains("reports")) {
```

Comment:
I do not understand what you are trying to achieve here.

Reply:
It was to please Coverage about the catch (*NOT_FOUND) I added around the deleteReport call

Reply:
So you can remove it when you will remove the `catch (HttpClientErrorException.NotFound e)` (see @SlimaneAmar comment) ?

Reply:
Done